services/hanke-service/src/main/resources/application.properties

```diff
@@ -23,6 +23,9 @@ info.java-version = ${java.specification.version}
 info.java-vendor = ${java.specification.vendor}
 server.forward-headers-strategy=FRAMEWORK
 
+#for rest api dateformat to work
```

Comment:
Voisi kommentoida myös että vaikuttaa ?-parametrien parsimiseen, ei siihen mitä JSONin sisällä on.

Reply:
Tuosta en osaa sanoa kyllä mitään. Ilman tuota ei toimi kriteerien antaminen päivämäärämuodossa.

Reply:
Siis, käsittääkseni tuo property vaikuttaa siihen kuinka kutsun .....?param=<date>  arvoja parsitaan, ei siihen kuinka yleisesti REST apissa JSONin payloadin sisältä parsitaan päivämääriä.  Eli tarkoitukseni oli että tuota "for rest api dateformat to work" olisi esim. "For REST API request parameter parsing to work for date values".

Reply:
Muutan kommentin noin :)